nextflow_schema.json

```diff
@@ -2,109 +2,130 @@
-            "description": "",
+            "description": "Parameters used by kraken2 to classify all reads provided.",
             "default": "",
             "properties": {
-                "single_end": {
-                    "type": "boolean"
-                },
-                "long_read": {
-                    "type": "boolean"
-                },
-                "enable_filter": {
-                    "type": "boolean"
+                "kraken2db": {
+                    "type": "string",
+                    "default": "https://genome-idx.s3.amazonaws.com/kraken/k2_standard_08gb_20231009.tar.gz",
+                    "description": "The database which is used in the classification step. For input how to use this parameter to fine-tune the step refer to usage/Databases/kraken2"
                 },
-                "filter_trimmed": {
-                    "type": "boolean"
+                "save_output_fastqs": {
+                    "type": "boolean",
+                    "description": "If the unclassified and classified reads should be saved to different files. The default false should be used",
+                    "hidden": true
                 },
-                "skip_blastn": {
-                    "type": "boolean"
+                "save_reads_assignment": {
+                    "type": "boolean",
+                    "default": true,
+                    "hidden": true,
+                    "description": "If the read assignments should be saved. Leave this to be true"
                 },
-                "filter_with_kraken2": {
-                    "type": "boolean"
+                "kraken2confidence": {
+                    "type": "number",
+                    "default": 0.05,
+                    "description": "Confidence in the classification of a read as a certain taxon. Refere to https://github.com/DerrickWood/kraken2/blob/master/docs/MANUAL.markdown#confidence-scoring for details."
                 }
             }
         },
         "kraken2taxa": {
             "title": "kraken2taxa",
             "type": "object",
-            "description": "",
+            "description": "Used for fine-tuning the isolation step of kraken2 output. Also here the taxon/taxa to be assessed/to be filtered can be set.",
             "default": "",
             "properties": {
+                "cutoff_tax2filter": {
+                    "type": "integer",
+                    "default": 2,
+                    "description": "If a read has less k-mers assigned to the taxon/taxa to be assessed/to be filtered the read is ignored by the pipeline."
+                },
+                "cutoff_unclassified": {
+                    "type": "number",
+                    "default": 0.0,
+                    "description": "If the ratio of the sum of k-mers mapped to the taxon/taxa to be assessed/to be filtered to unclassified is below this threshold the read is ignored by the pipeline. Can take values between 0.0 and 1.0.",
```

Comment:
In addition to 
`"description": "xyz"`
you can also use
`"help_text": "xyz"`
For more text that appears on demand, e.g. in https://nf-co.re/ampliseq/2.8.0/parameters#input click on "Help Text"